aries-site/src/examples/templates/FilterControls/FilterControls.js

```diff
@@ -41,19 +42,28 @@ export const FilterControls = ({
   }, [syncFilteredResults]);
 
   return (
-    <Box gap="xsmall" flex={false}>
-      <Box direction="row" align="end" gap="small">
-        {searchFilter && (
-          <SearchFilter placeholder={searchFilter.placeholder} />
-        )}
-        <Filters />
+    <Box flex={false} fill="horizontal">
+      <Box direction="row" align="start" justify="between" wrap>
```

Comment:
I still find the `wrap` to not look so good on my small laptop, as shown:

<img width="472" alt="Screen Shot 2021-07-15 at 2 54 42 PM" src="https://user-images.githubusercontent.com/11637956/125862836-f5354402-5c5d-4639-8f30-796689b88f03.png">

Can we remove the wrap here and ensure the SearchFilter is within a `<Box flex="shrink" />`?